Modules/Tokens/src/Tokens.Application/Infrastructure/Persistence/Repository/ITokensRepository.cs

```diff
@@ -9,8 +9,8 @@ namespace Backbone.Modules.Tokens.Application.Infrastructure.Persistence.Reposit
 public interface ITokensRepository
 {
     Task Add(Token token);
-    Task<Token> Find(TokenId tokenId);
-    Task<DbPaginationResult<Token>> FindAllWithIds(IEnumerable<TokenId> ids, PaginationFilter paginationFilter, CancellationToken cancellationToken);
+    Task<Token> Find(TokenId tokenId, IdentityAddress? address);
+    Task<DbPaginationResult<Token>> FindAllWithIds(IdentityAddress owner, IEnumerable<TokenId> ids, PaginationFilter paginationFilter, CancellationToken cancellationToken);
```

Comment:
This doesn't have to be the owner. It might as well be anyone else.

```suggestion
    Task<DbPaginationResult<Token>> FindAllWithIds(IdentityAddress activeIdentity, IEnumerable<TokenId> ids, PaginationFilter paginationFilter, CancellationToken cancellationToken);
```

Reply:
changes made